### ISSUE TITLE
On admin.php?page=gowp_crudescolas_criar

### ISSUE THREAD

**Issue by bigbabert** [georgemendonca/crudescolas#1]
WordPress database error: [Table 'google-bgbitaly.escola' doesn't exist]
SHOW FULL COLUMNS FROM `escola`

Notice: Undefined variable: message in /Applications/MAMP/htdocs/exemple.com/wp-content/plugins/crudescolas/estola-criar.php on line 14

The table must be manually entered in the db?

**Comment by georgemendonca** [comment]
Exactly, just copy the SQL code of the table creation script "school" and inserts (records) and add directly into the WordPress database.
If you have any questions just post here. ;)

![phpmyadmin](https://cloud.githubusercontent.com/assets/414884/7552608/e072aa9a-f694-11e4-9bf2-7daf05206f18.png)
